docs/setup/evss_aws_govcloud_proxy.md

```diff
@@ -0,0 +1,21 @@
+# Accessing the Reference Data Service
+### Swagger
+EVSS team has swagger docs up for their RefData service. Following are instructions for how to access them in your browser - basically you're going to do a setup similar to the socks proxy, but using the staging jumpbox since that's the environment that's connected to EVSS
+- `ssh -D 2002 -N ip-172-30-20-155.us-gov-west-1.compute.internal` - establishes a tunnel on local port 2002 to the staging jumpbox.
+- Configure browser proxy switcher with a new profile for SOCKS5 on localhost:2002, and set it up to use that proxy profile for host `internal-staging-services-1341723990.us-gov-west-1.elb.amazonaws.com`
```

Comment:
Same with the EVSS ELB, please remove this from the doc and we'll refer to it from elsewhere.